### PR TITLE
Ejercicios tema 2

### PR DESCRIPTION
**Pull request by Jefffer**
## Activity 2

This pull request includes the solution for Activity 2. There are two JavaScript files, `cinema.js` and `actividad.js`. Below is a summary of the most important changes:

### Cinema Seat Management:
* Introduced a function to initialize a matrix of seats with a specified size and initial state (`setup`).
* Added a function to suggest contiguous seats starting from the row furthest from the screen (`suggest`).

### Image Processing Operations:
* Added functions to convert images to different color scales: red, green, blue, grayscale, and black and white (`redConverter`, `greenConverter`, `blueConverter`, `greyConverter`, `blackAndWhiteConverter`).
* Implemented a function to scale down an image by removing every other row and column (`scaleDown`).
* Added functions to adjust image brightness and invert colors (`dimBrightness`, `invertColors`).
* Introduced a function to merge two images with specified alpha blending factors (`merge`).

These changes enhance the functionality of the cinema seat management system and provide a comprehensive set of image processing tools.

### CODE REVIEW

File: Tema_2/Jefferson_Rodriguez_Rodriguez/cinema/cinema.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    // Se recorren los asientos empezando por la ultima butaca de la fila más lejana.
+    for (let i = N-1; i >= 0; i--) {
+        let contador = 0;
+        // let asientosSugeridos = [];
+        for (let j = 0; j < N; j++) {
+            if (!butacas[i][j].estado) {
+                contador++;
+                asientosSugeridos.push(butacas[i][j].id);
+                if (contador === numeroAsientos) {
+                    return asientosSugeridos;
```
Comment (UnirCs) [inline Tema_2/Jefferson_Rodriguez_Rodriguez/cinema/cinema.js:45]: como comente en clase, no se aceptan soluciones que violen invariantes usando breaks, return o continue dentro de bucles.

**Review by UnirCs — CHANGES_REQUESTED** [review]
como comente en clase, no se aceptan soluciones que violen invariantes usando breaks, return o continue dentro de bucles.

File: Tema_2/Jefferson_Rodriguez_Rodriguez/cinema/cinema.js (hunk 1 above)
Comment (Jefffer) [reply Tema_2/Jefferson_Rodriguez_Rodriguez/cinema/cinema.js]: Corregido, me dices si algo no te cuadra :)

**Review by UnirCs — APPROVED** [review]
OK!

**Pull request merged**